### DISCUSSION TITLE
Import module to CLojureScript

### DISCUSSION THREAD

**Discussion by SergejGleithmann** [mdbootstrap/TW-Elements-React#170] [Support from community] [answered]
Hello everyone,

I have a problem when trying to import this module in ClojureScript:

```cljs
(:require
   [reagent.core :as r]
   [re-frame.core :as re-frame]
   [tw-elements-react :refer [TERipple]])
``` 
results in 
```
Closure compilation failed with 1 errors
; --- node_modules/tw-elements-react/dist/js/tw-elements-react.umd.min.js:13
; Dynamic import expressions cannot be transpiled.
```
Does someone know what Im doing wrong?

**Comment by iprzybysz** [comment]
Hi @SergejGleithmann, we can't help you with it because we don't have enough experience with CLojureScript. Maybe someone from the community will help you with this problem.

**Comment by SergejGleithmann** [comment, marked as answer]
I have found the solution on [StackOverflow](https://stackoverflow.com/questions/78477595/tailwind-elements-react-integration-into-clojurescript).

tl;dr When using shadow-cljs you need to use version `^2.26.6`. Furthermore you need to upgrade react to version `^18.0.0`. This will allow dynamic imports.
